database/initial-setup/populate_config.sql

```diff
@@ -5,7 +5,8 @@ VALUES
 ('winners', 3),
 ('commentsRequired', 1),
 ('pegWithoutKeyword', 0),
-('requireValues', 1);
+('requireValues', 1),
+('remoteLocationWeighting', 2);
```

Comment:
Mmm very true, but can this can be kicked down the road? This PR isn't really about that part of the functionality. Either way +1 to the idea, I like it